### ISSUE TITLE
v1.0.175 cannot launch on MacOS M1

### ISSUE THREAD

**Issue by aimuch** [Eugeny/tabby#6201]
**Describe the problem**:
[A clear and concise description of what the bug is.]
v1.0.175 cannot launch on MacOS M1
**To Reproduce**:
[Steps to reproduce the behavior]
```shell
❯ /Applications/Tabby.app/Contents/MacOS/Tabby; exit
innerError Error: Cannot find module '../build/Debug/pty.node'
Require stack:
- /Applications/Tabby.app/Contents/Resources/app.asar/node_modules/@tabby-gang/node-pty/lib/unixTerminal.js
- /Applications/Tabby.app/Contents/Resources/app.asar/node_modules/@tabby-gang/node-pty/lib/index.js
- /Applications/Tabby.app/Contents/Resources/app.asar/dist/main.js
-
    at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at Function.n._resolveFilename (node:electron/js2c/browser_init:245:1105)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (/Applications/Tabby.app/Contents/Resources/app.asar/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/Applications/Tabby.app/Contents/Resources/app.asar/node_modules/@tabby-gang/node-pty/lib/unixTerminal.js:33:15)
    at Module._compile (/Applications/Tabby.app/Contents/Resources/app.asar/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1169:10)
    at Module.load (node:internal/modules/cjs/loader:988:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Applications/Tabby.app/Contents/Resources/app.asar/node_modules/@tabby-gang/node-pty/lib/unixTerminal.js',
    '/Applications/Tabby.app/Contents/Resources/app.asar/node_modules/@tabby-gang/node-pty/lib/index.js',
    '/Applications/Tabby.app/Contents/Resources/app.asar/dist/main.js',
    undefined
  ]
}
objc[3687]: Class WebSwapCGLLayer is implemented in both /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebCore.framework/Versions/A/Frameworks/libANGLE-shared.dylib (0x2458971a8) and /Applications/Tabby.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libGLESv2.dylib (0x1093d93c8). One of the two will be used. Which one is undefined.
Error returned: <dictionary: 0x2111da530> { count = 1, transaction: 0, voucher = 0x0, contents =
	"XPCErrorDescription" => <string: 0x2111da6c8> { length = 18, contents = "Connection invalid" }
}
```

**Comment by llkoio** [comment]
yes，v1.0.175 cannot launch on MacOS M1

**Comment by aimuch** [comment]
> yes，v1.0.175 cannot launch on MacOS M1 zip and pkg my toplap info：macos 12.1 M1 max

I also have tried v1.0.174, cannot launch yet.

**Comment by iambudi** [comment]
Confirmed Tabby 175 can not launch on Mac. Reinstall 174 on top of 175 won't work, unless delete the Tabby app first before reinstall.

**Comment by Eugeny** [comment]
The fixed nightly build should be ready here soon: https://github.com/Eugeny/tabby/actions/runs/2189343264

**Comment by iambudi** [comment]
> The fixed nightly build should be ready here soon: https://github.com/Eugeny/tabby/actions/runs/2189343264 - please let me know how it goes - unfortunately I don't have an M1 machine to test on.
> 
> > <img alt="" width="48" src="https://repository-images.githubusercontent.com/77213120/49dde400-c8c2-11e9-812a-e1407f1f42af"><img alt="" height="14" src="https://camo.githubusercontent.com/b6a12909f1e31185a69a73d59208c507a992236d3230f9fc18e85058ae3d19e7/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f66617669636f6e732f66617669636f6e2e737667"> GitHub**[fixed ARM macOS builds - fixes #6201 · Eugeny/tabby@4a82294](https://github.com/Eugeny/tabby)**A terminal for a more modern age. Contribute to Eugeny/tabby development by creating an account on GitHub.

It works now.
![image](https://user-images.githubusercontent.com/31809529/164111471-37b17d44-913f-4335-98c4-6c34d33a0e47.png)

**Comment by Eugeny** [comment]
Thanks for confirming :+1:
